fedimint-core/src/admin_client.rs

```diff
@@ -269,7 +269,7 @@ mod serde_tls_cert {
     {
         let value: Cow<str> = Deserialize::deserialize(deserializer)?;
         Ok(rustls::Certificate(
-            Vec::from_hex(&value).map_err(D::Error::custom)?,
+            Vec::from_hex(&*value).map_err(D::Error::custom)?,
```

Comment:
Can you just do `value` here? De-referencing then borrowing is weird

Reply:
`value` doesn't work. However, I was able to replace the deref-borrow with `value.as_ref()`.